### DISCUSSION TITLE
4 pin board and LED?

### DISCUSSION THREAD

**Discussion by mcdent** [klaasnicolaas/home-assistant-glow#178] [Q&A] [answered]
This maybe a stupid question as I could not see a complete wiring diagram.
What is the LED RGB 5mm 4 pin and a Photodiode required for?
I was under the impression that the 4 pin board with photodiode picked up the flashes from the led and sent it to the ESP32 via the jumpers?
If so ,what's the other LED required for? Does it solder to the ESP32 for diagnostic / setup purposes?

Thanks

**Comment by klaasnicolaas** [comment, marked as answer]
The other led is for status / debugging purposes. For example, it will flash blue if there is no WiFi connection yet and with every detected pulse it will flash 1x red.

**Comment by akula3** [comment]
Whereas the green part of RGB led is wired, but has no function behind, see [here](https://github.com/klaasnicolaas/home-assistant-glow/discussions/177).
